src/Consolonia.Core/Text/GlyphTypeface.cs

```diff
@@ -26,24 +26,22 @@ public bool TryGetGlyphMetrics(ushort glyph, out GlyphMetrics metrics)
 
         public ushort GetGlyph(uint codepoint)
         {
-            checked
-            {
-                return (ushort)codepoint;
-            }
+            // we just return placeholder chars as we are pushing the raw
+            // text to the console
+            return 'X';
```

Comment:
As I understand character 21 exists for this purpose: https://www.ascii-code.com/21  :  there is an ASCII control character designed to indicate an error or negative acknowledgment, which is not meant to be printed by terminals. It's called the Negative Acknowledge character, abbreviated as NAK, and it has the ASCII code 21 in decimal (or 0x15 in hexadecimal).

    Name: Negative Acknowledge (NAK)
    ASCII Code: 21 (decimal), 0x15 (hexadecimal)
    Purpose: Used in communication protocols to indicate that a received message or transmission is invalid or contains errors.
    Display: It's a non-printable control character, so terminals typically do not display it.

In communication systems, the NAK character is sent from the receiver to the sender to signal that the data received was corrupted or not understood, prompting the sender to retransmit the message.

Example Usage in Protocols:

    File Transfer Protocols: NAK is used to request retransmission of data packets.
    Error Detection: Helps in simple error-handling schemes where acknowledgment is required.

Note: Since NAK is a control character and not a printable one, terminals and text displays do not render it visibly. Instead, it functions as a control signal within communication streams.

Summary: The ASCII Negative Acknowledge (NAK) character (code 21) indicates errors in communication and is not displayed by terminals.

Reply:
did you want me to update this?

Reply:
Just a hint, up to you